cylc/flow/network/graphql.py

```diff
@@ -95,8 +95,13 @@ def instantiate_middleware(middlewares):
 def null_setter(result):
     """Set type to null if result is empty/null-like."""
     # Only set empty parents to null.
-    if result in EMPTY_VALUES:
-        return NULL_VALUE
+    try:
+        if result in EMPTY_VALUES:
+            return NULL_VALUE
```

Comment:
I guess you're avoiding booleans here (by not just doing `if not result:...`)?

Reply:
`EMPTY_VALUES`  only contains `[]` and `{}`  as we want to be able to reset booleans and strings to `False`/`""` empties.